packages/components/src/components/pin-code/pin-code-utils.ts

```diff
@@ -0,0 +1,104 @@
+import type { FormState } from '../../utils/form/form-state';
+import type { Styles } from 'jss';
+import { consoleWarn, getTagNameWithoutPrefix, setAttributes } from '../../utils';
+
+export const PIN_CODE_TYPES = ['number', 'password'] as const;
+export type PinCodeType = (typeof PIN_CODE_TYPES)[number];
+
+export const PIN_CODE_LENGTHS = [4, 6] as const;
+export type PinCodeLength = (typeof PIN_CODE_LENGTHS)[number];
+
+export type PinCodeUpdateEvent = { value: string; isComplete: boolean };
+
+export type PinCodeState = FormState;
+
+export const hiddenInputSlotName = 'hidden-input';
+export const currentInputId = 'current-input';
+export const labelId = 'label';
+export const descriptionId = 'description';
+export const stateMessageId = 'state-message';
```

Comment:
Why are these shared/reusable utils? 
In other components we define these as local constants within `render()`. 
Please also check that names and values are identical to other components.